### PR TITLE
Upgrade to webdriverio@4

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs  @valentin-radulescu-hs can you review?

**Comment by adela-istrate-hs** [comment]
@flavius-tirnacop-hs can you review?

**Comment by flavius-tirnacop-hs** [comment]
LGTM

File: lib/adapters/webdriverio.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ function WebDriverIOAdaptor(webdriverioInstance) {
 
 WebDriverIOAdaptor.prototype = objectAssign({}, Browser, {
   takeScreenshot: function(callback) {
-    this._webdriverio.saveScreenshot(callback);
+    this._webdriverio.saveScreenshot()
+    .then(function(screenshot) {
```
Comment (valentin-radulescu-hs) [inline lib/adapters/webdriverio.js]: Indent this by two spaces

**Comment by valentin-radulescu-hs** [comment]
LGTM

**Comment by valentin-radulescu-hs** [comment]
LGTM, you can go ahead and merge this PR  🎉

**Pull request merged**